domains/linear-referencing/backend/src/LinearReferencingElements.ts

```diff
@@ -22,7 +22,7 @@ import {
 /** Base class for Spatial Location Element subclasses representing properties whose value is located along a Linear-Element and only applies to a portion of an Element.
  * @beta
  */
-export abstract class LinearlyLocatedAttribution extends SpatialLocationElement implements LinearlyLocatedAttributionProps, LinearlyLocatedBase {
```

Comment:
Note that "LinearlyLocatedBase" is not a "Props" interface, but a simpler interface that standardizes a convenient instance-access to the "getLinearElementId()" functionality (involving a query against the Db).

The "LinearlyLocatedBase" interface had been added to all LinearReferencing "element" classes: LinearlyLocatedAttribution, LinearLocationElement and ReferentElement. This PR is removing it from 2 of those 3 classes. IMO, such interface should be kept on those 3 since it is not a "Props" interface.

Reply:
Thanks. I guess I need to submit a PR to fix that.